packages/tracing-processing/src/models/db.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from "zod";
+import http from "http";
+
+const acceptedStatusCodes = Object.keys(http.STATUS_CODES).map((code) =>
+  parseInt(code, 10),
+);
```

Comment:
It can be simplified like this:

```suggestion
const acceptedStatusCodes = Object.keys(http.STATUS_CODES).map(Number);
```